flow/designs/asap7/mock-array-big/Element/config.mk

```diff
@@ -16,3 +16,5 @@ export DIE_AREA = $(shell python3 designs/asap7/mock-array-big/ce_die_area.py)
 export IO_CONSTRAINTS = designs/asap7/mock-array-big/Element/io.tcl
 
 export PDN_TCL = designs/asap7/mock-array-big/Element/pdn.tcl
+
+export MAX_ROUTING_LAYER = M5
```

Comment:
add a comment above as to why this is done? Is this required to get PDN rings to work or is it an additional improvement?